Bio/AlignIO/MafIO.py

```diff
@@ -119,8 +118,7 @@ def write_alignment(self, alignment):
 # Invalid function name according to pylint, but kept for compatibility
 # with Bio* conventions.
 def MafIterator(handle, seq_count=None, alphabet=single_letter_alphabet):
-    """
-    Iterates over lines in a MAF file-like object (handle), yielding
+    """Iterates over lines in a MAF file-like object (handle), yielding
```

Comment:
That isn't a self-contained one line summary. How about:

```python
"""Iterates over a MAF file handle as MultipleSeqAlignment objects.

Iterates over lines in a MAF file-like object (handle), yielding
MultipleSeqAlignment objects. SeqRecord IDs generally correspond to
species names.
"""
```

Also note final full stops (periods) on the one line summary and main text.